src/daemon/daemon.cpp

```diff
@@ -2053,33 +2053,37 @@ void mp::Daemon::install_sshfs(VirtualMachine* vm, const std::string& name)
 
     SSHSession session{vm->ssh_hostname(), vm->ssh_port(), vm->ssh_username(), key_provider};
 
-    mpl::log(mpl::Level::info, category, fmt::format("Installing sshfs in \'{}\'", name));
+    mpl::log(mpl::Level::info, category, fmt::format("Installing the multipass-sshfs Snap for \'{}\'", name));
```

Comment:
```suggestion
    mpl::log(mpl::Level::info, category, fmt::format("Installing the multipass-sshfs snap for \'{}\'", name));
```
Following https://snapcraft.io/docs at least…